### PR TITLE
minor doc improvements

### PR DESCRIPTION
**Pull request by thefourtheye**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -32,7 +32,7 @@
 
 ## Install
 
-```
+```bash
```
Comment (sindresorhus) [inline readme.md]: This is not correct as the below snippet is not valid Bash because of the `$`, but you could use `console`.

File: readme.md [hunk 2]
```diff
@@ -157,7 +157,7 @@ console.log(chalk.styles.red.open + 'Hello' + chalk.styles.red.close);
 - `red`
 - `green`
 - `yellow`
-- `blue` *(on Windows the bright version is used as normal blue is illegible)*
+- `blue` *(on Windows the bright version is used, as normal blue is illegible)*
```
Comment (Qix-) [inline readme.md]: The added comma is unnecessary.

Comment (thefourtheye) [inline readme.md]: Thanks for the review. Not a native English speaker. It didn't make sense to me without a punctuation mark.

Comment (Qix-) [inline readme.md]: Could be `...is used since normal blue...` (`since` instead of `as`). :)

**Comment by thefourtheye** [comment]
@sindresorhus @Qix- Addressed the comments. Thanks for the review :-)

**Comment by Qix-** [comment]
Thank you! :)

**Pull request merged**